crt_portal/static/js/ga_util.js

```diff
@@ -7,49 +7,47 @@ function gtag() {
   dataLayer.push(arguments);
 }
 
-function sendGAPublicClickEvent(event_name) {
-  gtag('event', 'click', { event_name: event_name });
+function getEventName(e) {
+  if (e.target.dataset.gaEventName) {
+    return e.target.dataset.gaEventName;
+  }
+  const ariaLabel = e.target.ariaLabel ?? 'unlabeled link element';
+  const innerText = e.target.innerText ?? 'empty link element';
```

Comment:
I like the use of the null coalescing operator `??`, but are you sure that's what you want here? Specifically, this will be blank if the attributes are `''`

```
'' ?? 'hello' === ''
'' || 'hello' === 'hello'
```